internal/integration/integration.go

```diff
@@ -212,10 +211,11 @@ func (s *Tester) createMachineUser(ctx context.Context, username string, userTyp
 	s.Organisation, err = s.Queries.OrgByID(ctx, true, s.Instance.DefaultOrganisationID())
 	logging.OnError(err).Fatal("query organisation")
 
+	notFoundErr := zerrors.ThrowNotFound(err, "QUERY-Dfbg2", "Errors.User.NotFound")
 	usernameQuery, err := query.NewUserUsernameSearchQuery(username, query.TextEquals)
 	logging.OnError(err).Fatal("user query")
 	user, err := s.Queries.GetUser(ctx, true, usernameQuery)
-	if errors.Is(err, sql.ErrNoRows) {
+	if errors.Is(err, notFoundErr) {
```

Comment:
`sql.ErrNoRows` is not longer returning as far as I can tell

Reply:
`sql.ErrNoRows` should be wrapped inside the NotFound error. `errors.Is` takes care of unwrapping underlying errors until this condition is satisfied or no more unwrapping can be done.

Please read https://pkg.go.dev/errors.

This change needs to be reverted.